cfme/tests/cloud/test_tenant.py

```diff
@@ -34,14 +37,21 @@ def test_tenant_crud(tenant):
     Metadata:
         test_flag: tenant
     """
+
     tenant.create(cancel=True)
     assert not tenant.exists()
 
     tenant.create()
     assert tenant.exists()
 
+    tenant.update({'name': 'my_new_name'})
+    refresh_provider(tenant.provider)
```

Comment:
**REQUIRED:** I was 100% sure I wrote a comment for this, but we don't need that ```refresh_provider``` function. it is a one liner. Just better to put ```tenant.provider.refresh_provider_relationships()``` instead.